proposals/collection-expressions-better-conversion.md

```diff
@@ -23,116 +23,85 @@ We add a new definition for ***better collection conversion from expression***,
 Given:
 - `E` is a collection expression with element expressions `[EL₁, EL₂, ..., ELₙ]`
 - `T₁` and `T₂` are collection types
-- `T'₁`:
-    - If `T₁` is a generic type, `T₁` with all type parameters unsubstituted
-    - If `T₁` is an array type, an unsubstituted array type
-    - Otherwise, `T₁`
-- `T'₂` is the same as `T'₁`, except for using `T₂` instead of `T₁`
 - `E₁` is the element type of `T₁`
 - `E₂` is the element type of `T₂`
 - `CE₁ᵢ` are the series of conversions from `ELᵢ` to `E₁`
 - `CE₂ᵢ` are the series of conversions from `ELᵢ` to `E₂`
 
-`T'₁` is at least as good as `T'₂` if there is an identity conversion between them.  
-`T'₁` is better than `T'₂` if:
-- `T'₁` is ReadOnlySpan<T>, and `T'₂` is not ReadOnlySpan<T>
-- `T'₁` is Span<T>, and `T'₂` is an *array_or_array_interface*
-- `T'₁` is implicitly convertible to `T'₂`, and `T'₂` is not implicitly convertible to `T'₁`.
-  
-Otherwise, `T'₁` neither as good as nor better than `T'₂`.
-
 If `E₁` and `E₂` have an identity conversion, then the element conversions are as good as each other. Otherwise, the element conversions to `E₁` are better than the element conversions to `E₂` if:
```

Comment:
```suggestion
If there is an identity conversoin from `E₁` to `E₂`, then the element conversions are as good as each other. Otherwise, the element conversions to `E₁` are better than the element conversions to `E₂` if:
```